app/components/footer_component.html.erb

```diff
@@ -1,37 +1,101 @@
+            <li><a class="link--black" href="/funding-and-support/tuition-fees">Tuition fees</a></li>
+            <li><a class="link--black" href="/funding-and-support/tuition-fee-and-maintenance-loans">Student finance</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">Teacher pensions</a></li>
+          </ul>
+        </div>
+        <div class="site-footer__links-column">
+          <h2 class="heading-m">Routes into teaching</h2>
+          <ul>
+            <li><a class="link--black" href="/train-to-be-a-teacher/if-you-have-a-degree">Train to teach with a degree</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Train to teach without a degree</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/teacher-degree-apprenticeships">Teacher degree apprenticeship</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/assessment-only-route-to-qts">Assessment only QTS</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/teaching-as-a-career/change-careers">Retrain as a teacher</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <hr class="site-footer__divider">
+    </section>
+
+    <div class="site-footer__social-icons">
+      <a href="https://www.facebook.com/getintoteaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
+        <%= helpers.fab_icon("facebook-f", "icon") %>
+        <span class="visually-hidden">Facebook - opens in a new tab</span>
+      </a>
+      <a href="https://www.instagram.com/get_into_teaching/" class="site-footer-top__social__link" target="_blank" rel="noopener">
+        <%= helpers.fab_icon("instagram", "icon") %>
+        <span class="visually-hidden">Instagram - opens in a new tab</span>
+      </a>
+      <a href="https://www.linkedin.com/company/9258520?trk=tyah&trkInfo=idx%3A1-1-1%2CtarId%3A1424345327269%2Ctas%3Aget+into+teaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
+        <%= helpers.fab_icon("linkedin-in", "icon") %>
+        <span class="visually-hidden">LinkedIn - opens in a new tab</span>
+      </a>
+      <a href="https://twitter.com/getintoteaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
+        <%= helpers.fab_icon("x-twitter", "icon") %>
+        <span class="visually-hidden">X, formerly known as Twitter - opens in a new tab</span>
+      </a>
+      <a href="http://www.youtube.com/user/getintoteaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
+        <%= helpers.fab_icon("youtube", "icon") %>
+        <span class="visually-hidden">Youtube - opens in a new tab</span>
+      </a>
     </div>
+
     <div class="site-footer-bottom">
       <div class="site-footer-bottom__links-container">
+        <h2 class="visually-hidden">Support links</h2>
```

Comment:
@spencerldixon This works really well but I think we need to move it above the social media icons (around line 74?) - could you possibly do that please?